### ISSUE TITLE
PyKEEN: Create demo that uses node embeddings

### ISSUE THREAD

**Issue by darabos** [lynxkite/lynxkite-2000#56]
The primary feature of PyKEEN is making node embeddings that allow good reconstruction of the graph structure. Make a demo where we use such embeddings in a downstream task.

- Pick a dataset. Let's say it's a social network with age/sex/location data.
- Pick a task. Let's say we want to predict if two people are married.
- Use PyKEEN to create node embeddings from the graph structure.
- Train an MLP for which the input is a concatenation of:
  - age/sex/location of person 1 encoded as a vector
  - PyKEEN embedding of person 1 encoded as a vector
  - age/sex/location of person 2 encoded as a vector
  - PyKEEN embedding of person 2 encoded as a vector

  And the output is the probability of being married. (For which we would need to have training data.)

  (Naturally the order of person 1 / person 2 should not matter. You could have the model process `p1` and `p2` separately for a few layers and then sum them: `y = MLPb(MLPa(p1) + MLPa(p2))`. The role of `MLPa` would be to process the `p1`/`p2` vectors so that addition preserves whatever we need to know about them.)

- Measure the performance of this model when trained on the full dataset vs when the age/sex/location data is corrupted vs when the edge data is corrupted. (By corrupted I mean we take a random permutation. So it has the same distribution, but it has zero predictive power.) To be safe, you can also measure the performance when both are corrupted, where we expect to see the performance of random guesses.

The downstream MLP model should use LynxKite's PyTorch model tools and this workspace should show how you can simply transfer embeddings from the PyKEEN boxes to the PyTorch boxes.

That said, maybe implement this in a notebook first (without LynxKite), so we are not chasing LynxKite bugs and data science bugs at the same time. If you can find a similar example that is already implemented then even better.

**Comment by Mackofi** [comment]
I have been working on this for a while in a notebook, got the PyKEEN side down. Trained a model that can recall graph structure with 90% accuracy and achieved over 99% f1 score. 
Only problem I'm currently having is how could I get training data that has marriages? Currently I am trying to synthetically generate it onto the original dataset with mixed results, but I'll keep working on it.
